src/index.ts

```diff
@@ -19,6 +19,7 @@ const VerifiedSnapVersionStruct = object({
 
 export const VerifiedSnapStruct = object({
   id: string(),
+  name: string(),
```

Comment:
There will potentially be more metadata about a snap, such as BIP paths and so on.

Since the main properties are used for allow list validation, all auxiliary data should live in a nested object for separation of what is used for security and what for informatory.
```suggestion
  metadata: object({ name: string() }),
```

Reply:
Good point! I've changed it.